Dockerfile

```diff
@@ -7,11 +7,15 @@ RUN go build -o bin/hiveutil github.com/openshift/hive/contrib/cmd/hiveutil
 RUN go build -o bin/hiveadmission github.com/openshift/hive/cmd/hiveadmission
 RUN go build -o bin/hive-operator github.com/openshift/hive/cmd/operator
 
-FROM registry.access.redhat.com/ubi7/ubi
+#FROM registry.access.redhat.com/ubi7/ubi
```

Comment:
Is the intention to go back to UBI at some point?

If not, should we just remove this line instead of commenting it out?

Reply:
Mixed on this, I'd kinda like it to be kept in our face a little.